modules/caddytls/acmeissuer.go

```diff
@@ -88,6 +88,15 @@ type ACMEIssuer struct {
 	// will be selected.
 	PreferredChains *ChainPreference `json:"preferred_chains,omitempty"`
 
+	// The validity period to ask the CA to issue a certificate for.
+	// Default: 0 (CA chooses lifetime).
+	// This value is used to compute the "notAfter" field of the ACME order;
+	// therefore the system must have a reasonably synchronized clock.
+	// NOTE: Not all CAs support this. Check with your CA's ACME
+	// documentation to see if this is allowed and what values may
+	// be used.
```

Comment:
```suggestion
	// be used. EXPERIMENTAL: Subject to change.
```

Reply:
I just did it. Sorry for the delay, I am quite busy at this moment with other work. I looked into using "Allow edits from maintainers" to give you access. But unfortunately it seems that this option is still not supported by GH for repo owned by org (which is the case here) : https://github.com/orgs/community/discussions/5634 :disappointed:

Reply:
Ohhh gotcha, no problem. Thanks for the patch! I just don't want to overpromise :)